response_operations_ui/views/admin.py

```diff
@@ -44,15 +53,48 @@ def update_banner():
     logger.debug("Updating banner", user=current_username())
     form = BannerAdminForm(form=request.form)
     banner = form.banner.data
-    if form.delete.data or not banner:
+    logger.debug("Banner update", user=current_username(), banner=banner)
+    admin_controller.set_banner_and_time(form.banner.data)
+    return redirect(url_for("admin_bp.remove_alert"))
+
+
+# Parser.parse allows the string returned from redis to be converted into a datetime object from string
+@admin_bp.route('/banner/remove', methods=['GET'])
+@login_required
+def view_and_remove_current_banner():
+    breadcrumbs = [{"text": "Banner Admin", "url": "/admin/banner"},
+                   {"text": "Setting Banner", "url": ""}]
+    logger.debug("Deleting alert", user=current_username())
+    current_banner = admin_controller.current_banner()
+    time_banner_set = admin_controller.banner_time_get()
+    time_banner_set = parser.parse(time_banner_set)\
+        .strftime('%d' + set_suffix(datetime.today().day) + ' %B ' + '%Y ' + 'at %H' + ':%M')
+    if current_banner:
+        return render_template('remove-alert.html',
+                               current_banner=current_banner,
+                               breadcrumbs=breadcrumbs,
+                               time_banner_set=time_banner_set)
+    else:
+        return redirect(url_for("admin_bp.banner_admin"))
+
+
+@admin_bp.route('/banner/remove', methods=['POST'])
+@login_required
+def remove_alert():
+    logger.debug("Updating banner", user=current_username())
+    form = BannerAdminForm(form=request.form)
+    banner_removed = ''
+    delete = form.delete.data
+    if delete:
+        session['banner_removed'] = banner_removed
         logger.debug("Banner deleted", user=current_username())
         admin_controller.remove_banner()
-    else:
-        logger.debug("Banner update", user=current_username(), banner=banner)
-        admin_controller.set_banner(form.banner.data)
     return redirect(url_for("admin_bp.banner_admin"))
 
 
-def remove_banner():
-    admin_controller.remove_banner()
-    return redirect(url_for("admin_bp.banner_admin"))
+def set_suffix(today):
```

Comment:
This one is an eyebrow raiser, im surprised that datetime doesn't support date suffixes. The stackoverflow post on this is from 9 years ago but i cant cant find any recent documentation. Could you briefly comment this and reference the stack overflow page https://stackoverflow.com/questions/5891555/display-the-date-like-may-5th-using-pythons-strftime